Smartling.API/Smartling.Api.csproj

```diff
@@ -7,6 +7,9 @@
     <Version>2.4.34.0</Version>
     <Authors>Smartling</Authors>
     <Copyright>Copyright © Smartling 2019</Copyright>
+    <SignAssembly>false</SignAssembly>
+    <AssemblyOriginatorKeyFile>key1.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>false</DelaySign>
```

Comment:
Once again - **sdk must not know about some keys** etc. Can you just open this code not in `trados plugin` solution but as a separate project? I think it's your ide generates this.